src/db/tripledb.plt

```diff
@@ -0,0 +1,23 @@
+:- begin_tests('db/tripledb').
+
+:- use_module(library('db/tripledb'),
+    [ tripledb_load/2,
+      tripledb_ask/3
+    ]).
+
+test('tripledb_load(URL)') :-
+  tripledb:tripledb_load('https://protege.stanford.edu/ontologies/pizza/pizza.owl',
```

Comment:
unit tests should run without Internet connection, so only local files may be used.

Reply:
Maybe it would be worth creating a small RDF file covering some different cases from the perspective of the triple DB